app/actions/article.js

```diff
@@ -11,36 +11,19 @@ export function fetchArticle(articleId) {
   };
 }
 
-export function fetchTopic(topicId) {
+export function postArticleHighlights(highlightsString, articleId) {
+  console.log(highlightsString, articleId)
   return (dispatch) => {
-    dispatch({ type: 'FETCH_TOPIC'});
+    dispatch({ type: 'POST_HIGHLIGHTS'});
 
-    return fetch(`http://localhost:5000/api/topics/${topicId}/?format=json`) // TODO: resolve this absolute URL issue with backend
+    return fetch(`http://localhost:5000/api/postHighlights/${articleId}`, {
```

Comment:
I'd create an issue for that. Can't say whether that's something that Babel will auto-polyfill.